cli/src/lib/validators/validatePackageExports.js

```diff
@@ -120,7 +194,7 @@ module.exports.validatePackageExports = async (
             return false
         }
         await fixPackage(pkg, expectedPackage, { paths })
-        valid = true
+        return true
     }
-    return valid
+    return !!valid
```

Comment:
The use of bitwise and assignment (`&=`) means that `valid` will be `1` instead of `true` and `0` instead of `false` and hence we must coerce `valid` to a boolean.

While this could be fixed by using logical and assignment (`&&=`), that operator is only supported in node v16+.